test/helpers/utils.go

```diff
@@ -296,13 +296,8 @@ func WriteOrAppendToFile(filename string, data []byte, perm os.FileMode) error {
 
 // DNSDeployment returns the manifest to install dns engine on the server.
 func DNSDeployment() string {
-	var DNSEngine = "kubedns"
-	k8sVersion := GetCurrentK8SEnv()
-	switch k8sVersion {
-	case "1.11", "1.12":
-		DNSEngine = "coredns"
-	}
-	return GetFilePath("provision/manifest/" + DNSEngine + "_deployment.yaml")
+	// dns_deployment.yaml will be created by the provision/k8s_install.sh
+	return GetFilePath("provision/manifest/dns_deployment.yaml")
```

Comment:
I'm not agree in this one. If someone runs the test with `--cilium.provision=false` the `/provision/manifest/dns_deployment.yaml` will be not in there at all, and the test will fail.